tests/boardfarm_plugins/boardfarm_prplmesh/tests/client_association_link_metrics.py

```diff
@@ -39,12 +31,11 @@ def runTest(self):
         controller.ucc_socket.dev_send_1905(agent.mac, 0x800D,
                                             tlv(0x95, 0x0006, '{sta_mac}'.format(sta_mac=sta_mac)))
         self.check_log(agent,
-                       "client with mac address {sta_mac} not found".format(sta_mac=sta_mac),
-                       timeout=30)
+                       "client with mac address {sta_mac} not found".format(sta_mac=sta_mac))
         time.sleep(1)
 
         debug('sta: {}'.format(sta.mac))
-        sta.wifi_connect(agent.radios[0].vaps[0])
+        sta.wifi_connect_check(agent.radios[0].vaps[0])
```

Comment:
Perhaps a better name should be `sta.connected()->bool`

Reply:
No, because it's a combination of connecting, checking if it succeeded, and retrying.